data/korath/korath hails.txt

```diff
@@ -0,0 +1,1470 @@
+		"ritfaki"            # enjoy
+		"fenahac"            # devour
+		"anipar"             # stream
+	word
+		" "
+	word
+		"Ter Mite Bee Riot"        # There Might Be Riots
+		"Rasi Imarifan"                # Laminated Fish
+		"Rivriv Apakrek Binam Airot"   # Ringworld Crusaders
+		"Ragas-ragas it Ikup"          # Comrades on Toast
+		"Imoni Ranafpures"             # Illusion of Normalcy
+		"Apako Alpom"                  # North of Heaven (probably ironically)
+		"Pafam Ralret Apav Lrad Dkyir" # Cryptic in a Nifty Way
+		"Lrapu Lrad Hatof Pasabpek"    # Flammable People
+		"Ranabeses Iacfam Ikaf"        # Immunity to Death
+		"Raiakrabfec Svranc"           # Planck Massacre
+
+# This is probably not what Korath actually eat, but it's what I
+# imagine them eating if they were in our place in history on Earth.
+
+phrase "exile: eat good food"
+	word
+		"ila'ikri"                     # digest
+		"rameref"                      # ingest
+		"i'fo'rutnref"                 # consume
+		"bapamef"                      # devour
+		"ikaftir"                      # enjoy
+		"rataf"                        # dine on
+		"pagronref"                    # munch on
+		"raofavleb"                    # banquet
+		"pagronref"                    # masticate
+		"kininnref"                    # nibble
+	word
+		" "
+	phrase
+		"exile: good food"
+
+phrase "efreti: eat good food"
+	word
+		"ipamikri"                     # digest
+		"raneref"                      # ingest
+		"imfomruslref"                 # consume
+		"cahanef"                      # devour
+		"ikafsir"                      # enjoy
+		"rasaf"                        # dine on
+		"hadrolref"                    # munch on
+		"raofawpec"                    # banquet
+		"hadrolref"                    # masticate
+		"kilillref"                    # nibble
+	word
+		" "
+	word
+		"atanem oawih aok palem"       # fresh dark green lettuce
+		"lrabfos satab hadrep"         # crunchy rhino beetles
+		"ipebupkm palem lrad lrakaf"   # ripe succulent strawberries
+		"tuv lrallrac ralret lragab hikoc"  # baked cod with garlic
+		"kenniy enum lrallrac"         # baked sole fillet
+		"mmab koan inihv"              # Chilean Sea Bass
+		"rasi it maka makpes"          # fish en papillote
+		"runef robfe otaf"             # honeydew melons
+		"neca akmec lrad hadrep"       # crisp gala apples
+		"oban lriros lravriv monah"    # finely chopped yellow squash
+		"igam oawih kopac"             # shredded collard greens
+		"kano cotih lrad kainellref"   # live squirming mealworms
+		"soslrafem arpan lrivav lranel" # a bowlful of hornworm larvae
+		"hoponem irunus kofem"         # an entire ant colony
+		"rasocfok pameb lrivav rinin"  # a big heaping pile of waxworms
+		"lrapam ponek lropob akro"     # a nest of ostrich eggs
+		"idab mosik rilrit ralret ramaih" # chilled baby mice with garnish
+		"akmac pano nives ralret umsab lravriv" # a pasta of small snakes with mice meatballs
+
+phrase "exile: Recently"
```

Comment:
I feel like this whole section may be running afoul of the sense of time stuff? @ravenshining ?